DPNUnet.imjoy.html

```diff
@@ -470,7 +473,7 @@
             api.alert('No model available for exportting.')
 
 
-    async def predict(self):
+    async def predict_default(self):
```

Comment:
Instead of make `predict_default` independent from `predict`, I suggest to call `predict` inside `predict_default`.

This can remove duplicated code and also give a basic example on how should `predict` should be called.

Reply:
Good point. will be considered in next commit.